ert_shared/ensemble_evaluator/config.py

```diff
@@ -6,7 +6,7 @@
 logger = logging.getLogger(__name__)
 CLIENT_URI = "client"
 DISPATCH_URI = "dispatch"
-DEFAULT_PORT = str(8765 + random.randint(0, 30))
+DEFAULT_PORT = str(8765 + random.randint(0, 100))
```

Comment:
This should be done in some other way. Unix already offers `0` as a way to get a random port.

Reply:
Yes, and also this is not the problem it turns out :sweat_smile: